test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/cdc/CDCE2EIT.java

```diff
@@ -0,0 +1,230 @@
+        }
+        getIncreaseTaskThread().join(10000);
+        List<Map<String, Object>> actualProxyList;
+        try (Connection connection = getProxyDataSource().getConnection()) {
+            ResultSet resultSet = connection.createStatement().executeQuery(String.format("SELECT * FROM %s ORDER BY order_id ASC", getOrderTableNameWithSchema()));
+            actualProxyList = transformResultSetToList(resultSet);
+        }
+        Awaitility.await().atMost(10, TimeUnit.SECONDS).pollInterval(2, TimeUnit.SECONDS).until(() -> listOrderRecords(getOrderTableNameWithSchema()).size() == actualProxyList.size());
+        List<Map<String, Object>> actualImportedList = listOrderRecords(getOrderTableNameWithSchema());
+        assertThat(actualProxyList.size(), is(actualImportedList.size()));
+        SchemaTableName schemaTableName = getDatabaseType().isSchemaAvailable()
+                ? new SchemaTableName(new SchemaName(PipelineBaseE2EIT.SCHEMA_NAME), new TableName(getSourceTableOrderName()))
+                : new SchemaTableName(new SchemaName(null), new TableName(getSourceTableOrderName()));
+        PipelineDataSourceWrapper targetDataSource = new PipelineDataSourceWrapper(StorageContainerUtil.generateDataSource(getActualJdbcUrlTemplate(DS_4, false), getUsername(), getPassword()),
+                getDatabaseType());
+        PipelineDataSourceWrapper sourceDataSource = new PipelineDataSourceWrapper(generateShardingSphereDataSourceFromProxy(), getDatabaseType());
+        StandardPipelineTableMetaDataLoader metaDataLoader = new StandardPipelineTableMetaDataLoader(targetDataSource);
+        PipelineTableMetaData tableMetaData = metaDataLoader.getTableMetaData(PipelineBaseE2EIT.SCHEMA_NAME, "t_order");
+        PipelineColumnMetaData primaryKeyMetaData = tableMetaData.getColumnMetaData(tableMetaData.getPrimaryKeyColumns().get(0));
+        ConsistencyCheckJobItemProgressContext progressContext = new ConsistencyCheckJobItemProgressContext("", 0);
+        SingleTableInventoryDataConsistencyChecker checker = new SingleTableInventoryDataConsistencyChecker("", sourceDataSource, targetDataSource, schemaTableName, schemaTableName,
+                primaryKeyMetaData, metaDataLoader, null, progressContext);
+        DataConsistencyCheckResult checkResult = checker.check(new DataMatchDataConsistencyCalculateAlgorithm());
+        System.out.println(checkResult);
+    }
+    
+    private void registerStorageUnit() throws SQLException {
+        String registerStorageUnitTemplate = REGISTER_STORAGE_UNIT_SQL.replace("${user}", getUsername())
+                .replace("${password}", getPassword())
+                .replace("${ds0}", appendExtraParam(getActualJdbcUrlTemplate(DS_0, true)))
+                .replace("${ds1}", appendExtraParam(getActualJdbcUrlTemplate(DS_1, true)));
+        addResource(registerStorageUnitTemplate);
+    }
```

Comment:
Could we extract single REGISTER STORATGE UNIT in `PipelineBaseE2EIT`? So we could invoke it any times in sub classes, but not hard-code it

Reply:
OK